src/components/GameItem.js

```diff
@@ -16,14 +28,26 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
+
+    // Change css if spotted Item is litter
+    let spottedItemStyle  = " " 
+    if (this.state.spottedItem ) { 
+      this.props.type === "litter" ? spottedItemStyle = " spotted-litter" : spottedItemStyle = " spotted-nature";
```

Comment:
Nice!

I'm happy with this syntax. In case you wanted an alternative syntax, I might do this...

```javascript
let spottedItemStyle;
if (this.state.spottedItem) {
  spottedItemStyle = this.props.type === "litter" ? " spotted-litter" : " spotted-nature";
}
```

Note two things:
- Instead of giving a default value of `" "` to `spottedItemStyle`, since that doesn't really matter in this case (It will be re-assigned no matter what), just declaring the variable, so `let spottedItemStyle;`... this is a kind of common thing in JS
- Just another way to do ternaries-- since you're going to be assigning a new value to `spottedItemStyle` in both the true and the false cases, you can pull out that bit and put the ternary in the right-hand assignment side for less typing